rdr_service/app_util.py

```diff
@@ -389,3 +391,15 @@ def datetime_as_naive_utc(value):
 
 def is_care_evo_and_not_prod():
     return GAE_PROJECT != "all-of-us-rdr-prod" and get_account_origin_id() == "careevolution"
+
+
+def install_rate_limiting(app):
+    cache_location = config.getSettingJson('cache_storage_location', default='memory://')
+    default_rate_limit = config.getSettingJson('default_rate_limit', default='15/second')
+    Limiter(
+        app,
+        key_func=lambda: get_oauth_id() or get_remote_address(),
```

Comment:
Big fan of `lambda` functions, but not sure it's needed here, the or statement should suffice if a result is returned from either of those methods.
key_func=`get_oauth_id() or get_remote_address()`

Reply:
The `key_func` parameter shouldn't be the result of the methods, but instead needs to be a callable that gives the result of the or-statement. `get_oauth_id` and `get_remote_address` wouldn't actually be called here when setting up the Limiter, but instead are part of the definition of the function given as the `key_func` parameter.